core/src/main/java/org/infinispan/functional/Param.java

```diff
@@ -126,14 +126,20 @@ public static LockingMode valueOf(int ordinal) {
    enum ExecutionMode implements Param<ExecutionMode> {
       /**
        * Command is executed on its owners, in transactional mode in the context, too, but there it is not persisted.
+       * The result of the command is backed up on all sites configured for backup.
        */
       CLUSTER,
       /**
        * Command is executed only locally, it is not sent to remote nodes. If the command is a write and this node
        * is not an owner of given entry, the entry is not stored in the cache. If the command reads a value and
        * the entry is not available locally, null entry is provided instead.
        */
-      LOCAL;
+      LOCAL,
+      /**
+       * Command is executed only in the current site (same as {@link #CLUSTER}, but it is not sent for backup
+       * to other sites)
+       */
+      SITE_ONLY;
```

Comment:
I'll rename `CLUSTER` to `ALL`, we can mess with this a bit due to the `@Experimental` flag. It's not used anywhere, as it's the default anyway. And `LOCAL_SITE` sounds good.

I'll add some backdoor to the documentation of `CLUSTER` :)